Da huge stonking cans update. Adds a new bulky reagent storage vessel, makes botanists pay for advanced fertilizer.

Krysonism opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

This PR introduces a new type of reagent container, the plastic jerrycan.
The new cans can hold up 200 units, are bulky and spill / splash proof. 

They, like a condiment bottle, are able to be renamed and transfer small amounts of reagents as required in the botany lab.

The new cans can be printed from the med and service lathes after biotech has been researched.

The plant nutrients from the nutrimax are no longer free and are delivered in these new cans. 

Ez-nutrient bottles are still provided free of charge.

As these cans are not a beaker subtype they do not fit in any grenade casing.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

![newnutrients](https://user-images.githubusercontent.com/49783092/72782956-7ace0500-3c25-11ea-96de-620b0e192655.PNG)

## Why It's Good For The Game
This PR  facilitates the grinding, mixing, sale and transport of bulk reagents as well as solving some hydroponics related gripes.

The little bottles of hydroponics nutrients do not last very long, need to be set down careful to prevent spilling, get lost and create trash.

Since there is now an incentive to store reagents outside the backpack, theft of reagents can now occur.

If you don't want to pay for nutrients, you can still dispense them from the biogen or use free nutrients like nutriment and ash.

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added plastic jerrycans to the service / med protolathes under the biotech node.
balance: The nutrients from the Nutrimax now comes in 200u cans.
balance: Plant nutrients from the Nutrimax are no longer free for botanists. Ez-nutrient is still free.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Krysonism committed:
Adds cans to techwebs and nutrimax.

Shadowflame909 commented:
![](https://i.ytimg.com/vi/gtr_MFno130/hqdefault.jpg)

Botany is the nightmare of living in a capitalist society personified.

SoS please handle me with care.

dr-bee commented:
Botany at its current wages already have to pay for way, way too many things with no way to make money besides passive regen. So I would keep fertilizer free but have it only be in stocks of 1.

Krysonism commented:
> 
> 
> Botany at its current wages already have to pay for way, way too many things with no way to make money besides passive regen. So I would keep fertilizer free but have it only be in stocks of 1.

I will be looking at rebalancing and reworking the different nutrients. 

We should keep in mind that all 3 vendor nutrients can be dispensed from the biogen and that very good nutrients can be obtained for free by composting veggies or burning paper.

anconfuzedrock commented:
perhaps a botanist could consider not buying the fertilizer if they cannot afford it.

XDTM commented:
I'm out of the loop, why do botanists have to pay out of pocket for their own job supplies?

AdamElTablawy commented:
how exactly is botany supposed to fertilize their first plants without nutrient? is the idea to demand chemistry make them ash and stuff as well as mutagen?

kriskog commented:
There should be a free no effect fertilizer available. Imagine you walk into unused botany at 45 minutes, and all the trays are out of nutrient.

Shadowflame909 commented:
tfw can't afford nutrient

No nutrient means no plant

Solution is use biogen.

No plant for biogen.

No biogen no nutrient.

Botany forever suffering

cacogen commented:
look i'm not against these ugly clear opaque "jerrycans" being merged because i agree the small botany bottles are annoying for the reasons you described but their highest "resolution" as cobby has termed it in the past of 1u per transfer makes no sense given the volume/unwieldiness of a jerrycan and why do you have to pay for them

81Denton commented:
Correct me if I'm wrong, but would this make botanists the only job that has to use their own paycheck for essential job supplies?
Making them pay for fertilizer would be the same as making engineers pay for tools or making sec pay for their stun baton and handcuffs.

Krysonism commented:
> 
> 
> how exactly is botany supposed to fertilize their first plants without nutrient? is the idea to demand chemistry make them ash and stuff as well as mutagen?

There seems to be some confusion about game mechanics. The trays start with full nutrition and burning one paper bin yields 900u ash, a very good fertilizer that both heals and kills weeds.

>There should be a free no effect fertilizer available. Imagine you walk into unused botany at 45 minutes, and all the trays are out of nutrient.

This isn't the worst idea, i will consider doing this, although there are plenty of ways around this already.

You could spend 200 credits on a can of EZ-nutrient, compost the roundstart veggies, use blood & pest spray, grow harebells to use as a shitty fertilizer or burn paper.

>Correct me if I'm wrong, but would this make botanists the only job that has to use their own paycheck for essential job supplies?

> the fertilisers are a basic necessity for the job given freely everywhere else players can grow things. they aren't "luxury" items as cob coincidentally also termed strange seeds, etc.

The cans themselves are the luxury, the nutrients are still available free from a myriad of sources.

kriskog commented:
> > There should be a free no effect fertilizer available. Imagine you walk into unused botany at 45 minutes, and all the trays are out of nutrient.
> 
> This isn't the worst idea, i will consider doing this, although there are plenty of ways around this already.
> 
> You could spend 200 credits on a can of EZ-nutrient, compost the roundstart veggies, use blood & pest spray, grow harebells to use as a shitty fertilizer or burn paper.

EZ-nutrient is not effect free. It mutates your plants.
Your alternate suggestions here is you gatekeeping shit again. Lets assume someone joins botany with a few hours of game time, do you expect them to know about all this? Yes they are neat things you can do that people should feel good about figuring out with time, but we should strive to keep things available to people without having to bury it in an examine text or a wiki article. This is doubly true for easy jobs geared towards newer players.

You are also talking about using the cans as the incentive, but there is no obvious alternative to incentivize this against. I will not be okay with this unless there is a free easy to use alternative.


I am also not blind to the fact that you are introducing a fairly cheap 200u beaker that can be used in chem dispenser and also give you finer macro granularity. I am not yet sure what to think about that.

81Denton commented:
>The cans themselves are the luxury, the nutrients are still available free from a myriad of sources.

The three fertilizers are no luxury since all the actually fun botany content is locked behind mutations. If anything, they're the only alternative to unstable mutagen which relies on shitty interactions like begging for mutagen, stealing a chem dispenser or building your own.

Krysonism commented:
> EZ-nutrient is not effect free. It mutates your plants.
 What? It is the basic ass nutrient that resets the tray mutmod to 1, the default value. If that isn't side effect free, no side effect free nutrient exist in the code.

A solo botanist, who is so new that they don't know how to compost yet, is going to have no qualms with spending 200 credits on a can of Ez-nutrient.

XDTM commented:
>I am also not blind to the fact that you are introducing a fairly cheap 200u beaker that can be used in chem dispenser and also give you finer macro granularity. I am not yet sure what to think about that. At the very least you should not be able to build these, or you invalidate the x-large and metamaterial beakers.

I think large reagent tanks are fine, as long as they are bulky and cannot be fit into grenades. If it's still an issue they could be made too large to store in backpacks, making them less desirable than a normal large beaker.

>A solo botanist, who is so new that they don't know how to compost yet, is going to have no qualms with spending 200 credits on a can of Ez-nutrient.

No other job requires spending credits for a very basic job tool. Credit purchases should be for specialistic, advanced, or more efficient resources. 
You could define jerrycans as the advanced version of bottles, but i think it'd be fine if we gave it roundstart to botanists, all that would change is having less inventory fiddling when doing their basic job task.
Instead, i think that adding a limited amount of mutagen to the vendor for credits would be a nice idea, since it would let beginners get acquainted with the mutation mechanic a bit, while still requiring chemistry cooperation/thievery for full mutagen powergaming.

BadSS13Player commented:
Why not retain a few free bottles of each?

Krysonism commented:
> I think large reagent tanks are fine, as long as they are bulky and cannot be fit into grenades. If it's still an issue they could be made too large to store in backpacks, making them less desirable than a normal large beaker.

Just to clarify, this is already the case. Weight class bulky items are unable to be stored in backpacks.

I would even be willing to go so far as to prevent non-beakers / bottles from being loaded into a chem dispenser if the maintainers agree.

>Why not retain a few free bottles of each?

I will retain 3 bottles of ez-nutrient.

ExcessiveUseOfCobblestone commented:
> What? It is the basic ass nutrient that resets the tray mutmod to 1, the default value. If that isn't side effect free, no side effect free nutrient exist in the code.

mutmod 1 is not mutation-free, you're looking for mutmod 0. That said, I do NOT want a mut-free chem like robust harvest as our basic chem because mutations work both ways and disrupt the gene machine manipulation which I think is good. Robust in actuality should probably be much more expensive than it is now since it is very safe statwise.

Botany should start with just a couple of bottles of EZ (free) and have to use the biogen to get more so you're actually incentivized to make plants for plants instead of rushing your instakill face-melting tomatoes or your omega-healing wheat.

As for the cans, i'm concerned about them being used as 200u beakers in applications outside of botany. Also being able to transfer 1u from a can without the help of another more precise tool is kinda yucky, really should clamp down on the reagent precision of containers some day.

Krysonism commented:
> > What? It is the basic ass nutrient that resets the tray mutmod to 1, the default value. If that isn't side effect free, no side effect free nutrient exist in the code.
> 
> mutmod 1 is not mutation-free, you're looking for mutmod 0. 

I've never said that Ez-nutrient is mutation free, my point was that Ez-nutrient is the basic nutrient and doesn't increase mutations above the default level. 

>Also being able to transfer 1u from a can without the help of another more precise tool is kinda yucky, really should clamp down on the reagent precision of containers some day.

I am not married to that kind of precision, but i felt it was necessary for the cans to be an attractive alternative to the condiment bottles which I loathe.

FantasticFwoosh commented:
Its honestly cheaper to take up chain-smoking and use the cigarette ash scooped up to feed your first water-melon plant then forget the jerrycan's even exist with efficient biogenerator returns. Get good armchair code designers.

 I only want jerrycans in the game for portable welding tanks and spilling propellant/floor liquids like lube all over the place without emptying on single use or relying on a spray.

ExcessiveUseOfCobblestone commented:
> I've never said that Ez-nutrient is mutation free, my point was that Ez-nutrient is the basic nutrient and doesn't increase mutations above the default level.

I was trying to clarify in that conversation without quotedumping everyone.

If we're adding bottles on top of the canisters shouldn't be an issue to just pour can into bottle for higher precision. I like that more than 1u canisters.

Yeah we'll need something that can filter putting things in chem machinery. While funny, these prob shouldn't be going into IV drips or grinders either.

necromanceranne commented:
Hey uh just as a reminder, composting lets botanists add nutrients to trays so they can always reasonably fertilize their plants. This seems fine to me. Struck me as kinda weird composting isn't the first go-to for botanists anyway.

ATH1909 commented:
Friendly reminder that if you want your plants to NOT mutate their stats (such as potency, production speed, and yield) on their own, you HAVE to dose them with at least 1u of robust harvest at at least one point. Robust harvest is an _essential_ item for gene modding botanists, because without it, your super plants' stats will start to degrade and you won't be able to ensure that all of your plants' "fruits" will have equal amounts of chems in them. "Just use ash lmao" is NOT a solution; don't pretend like robust harvest doesn't have priority over even strange seeds for gene modding botanists. If you make it a premium item, botanists like me won't stop using it; we'll just have to add "beg the roboticists for cash" back to our roundstart routines again.

In short, this is the equivalent of making insuls a premium item for engineers. They don't _technically_ need them, but if you want to get anything complex done with gene modding, you NEED robust harvest.

> and disrupt the gene machine manipulation which I think is good.

... Fuck, us gene manipulators are screwed, aren't we?

Iamgoofball commented:
Uh, plant nutrients are literally a required thing for botanists to do their job. Hence why it's free to botanists.

Please don't merge this.

ATH1909 commented:
balance: The nutrients from the Nutrimax now comes **in 200u cans.**

bruh

So you're telling me that this PR makes it harder for newer botanists who don't know the roundstart botany powergaming tricks to get fertilizer for their plants _and_ adds a roundstart 200u beaker container that chemists can run over to the vending machines outside botany and buy?

... Why exactly are you doing this again?

> This PR facilitates the grinding, mixing, sale and transport of bulk reagents

by powergaming chemists, yes

> as well as solving some hydroponics related gripes.

who the fuck was complaining about botany's access to fertilizer?

> The little bottles of hydroponics nutrients do not last very long, need to be set down careful to prevent spilling, get lost and create trash.

Oh, woe is me, I have to press "q" (or left click on a table) to put down my fertilizer bottles instead of chucking them across the room with "r"! And the spent bottles add something for the janitor to clean up! We can't be having that, now, can we?

> Since there is now an incentive to store reagents outside the backpack, theft of reagents can now occur.

Ah yes, I, Snidely Whiplash, can finally steal the nutriment of botanists that I could have just bought myself from a vending machine in front of botany! I will use it for nefarious deeds, such as... growing plants*! Ahahahahahahahaaaaaaa! Stealing the fertilizer cans of botanists definitely won't be considered tiding or self-antagging!

*(and powergaming with chemistry, but chemists have a large enough paycheck to just buy a can of fertilizer themselves)

> If you don't want to pay for nutrients, you can still dispense them from the biogen or use free nutrients like nutriment and ash.

Then why the fuck did you make fertilizer cans premium items and remove the normal bottles of fertilizer from the vending machines? If you had kept the normal bottles of fertilizer in the vending machines and added these cans as a "convenience tool", I could sort of get why you'd do that, but this PR helps no one and just adds more annoyance to the lives of botanists.

CRITAWAKETS commented:
I wont go over how this is like shooting botanists in the knees for nothing, but things people have not yet said about this PR.

By the way you didnt show the prices of Ammonia and Diethylamine, probably because you set them ludicrously high yet also didnt make diethyl more expensive despite being better for everything but yield.
>Ammonia: 1000
>Diethylamine: 1000

>Ammonia can description:
>"A large container labled 'NT-Ag' liquid ammonia. A warning label reads: Store separately from chlorine-based cleaning products!"

This could be interpreted as promoting deadly chemical experiments and/or terrorism.

> This PR introduces a new type of reagent container, the plastic jerrycan.
> The new cans can hold up 200 units, are bulky and spill / splash proof.

> The new cans can be printed from the med and service lathes after biotech has been researched.

Golly gee, what do these cans cost? If you thought "more than a metamaterial beaker"...
>Plastic: 4000

You were wrong. It only costs plastic for something better than metamaterial and yet only requires biotech 1, a tech that can be obtained for 2500 points roundstart and is very often researched early.

Krysonism commented:
Putting the roundstart fruit or one harvest of plants into the Biogen gives you enough robust harvest to last like  the entire round. The fact that people are freaking out just shows us that this feature is never used.

> If you thought "more than a metamaterial beaker".

Why would you think that, these are bulky and cannot be put in grenades

>didnt make diethyl more expensive despite being better for everything but yield.

Ammonia is way better than diethylamine. Plant healing and nutrition points are not valuable, yield increase is. If anything diethylamine should be less expensive.

CRITAWAKETS commented:
> Ammonia is way better than diethylamine. Plant healing and nutrition points are not valuable, yield increase is. If anything diethylamine should be less expensive.

Do not make PR's to severely rebalance jobs without actually playing them.
If you actually played botany you would have known that is it important when you dont yet have earthsblood. (Though it is better to use ammonia when you do have blood of the earthy variety.)

Krysonism commented:
> > Ammonia is way better than diethylamine. Plant healing and nutrition points are not valuable, yield increase is. If anything diethylamine should be less expensive.
> 
> Do not make PR's to severely rebalance jobs without actually playing them.
> If you actually played botany you would have known that is it important when you dont yet have earthsblood. (Though it is better to use ammonia when you do have blood of the earthy variety.)

I do play botany thank you very much. I am just big brained enough to use the 900u of free ash provided by a paper bin to heal my plants instead of paying money like diethylamine like a chump.

cacogen commented:
you could update the billy bob paper with some of the meta strategies you described here and put it in hydroponics if you're intent on reducing players' access to roundstart fertiliser through the vending machines. that paper's been the same since like 2010

Shadowflame909 commented:
This would be fine if botany had money

Cobblestone?

Hello???

ExcessiveUseOfCobblestone commented:
you don't even need le epic paper strat, just having a plant that's used to build up your biogen funds will provide you with more than enough nutrients for your needs. People are throwing out the "do you even play" meme here but it seems like we're at a misunderstanding regarding very basic botany mechanics. Yes, you need SOME starting nutri, EZ being the starter. Do you need 30 x 3 (x 2) _of each type_ roundstart? No.

i lol'd at 
>This could be interpreted as promoting deadly chemical experiments and/or terrorism.

cacogen commented:
>it seems like we're at a misunderstanding regarding very basic botany mechanics

update billy

ATH1909 commented:
> you don't even need le epic paper strat, just having a plant that's used to build up your biogen funds will provide you with more than enough nutrients for your needs. People are throwing out the "do you even play" meme here but it seems like we're at a misunderstanding regarding very basic botany mechanics. Yes, you need SOME starting nutri, EZ being the starter. Do you need 30 x 3 (x 2) _of each type_ roundstart? No.
> 
> i lol'd at
> 
> > This could be interpreted as promoting deadly chemical experiments and/or terrorism.

But we do agree that this PR is a poor implementation of this idea, right? I mean, do you really think _anyone_ is gonna buy a _30u_ bottle of ammonia for _1,000 credits_?!

Also, I should specify that when I referred to "chemistry powergaming" earlier, I didn't mean "grenades", I meant "making chems early in the round that normally requires a >100u container to make without extra steps or an upgraded dispenser". These fertilizer cans can make some chems one-macro-able 5 minutes into the round, and that's something to take a closer look at (unless these can't be put in chem dispensers?).

ATH1909 commented:
Maybe we all have different views on this because we play botanist very differently from each other? I, for one, almost never go for ambrosia gaia, even though some other people swear by it. I make a dispenser instead of stealing one, and will generally focus on making one super meme plant from the traits I find in strange seeds instead of several lesser meme plants. I never buy or use any fertilizers other than robust harvest unless I'm pumping my plant's stats, and even then, I make sure to apply at least a 1u dose of robust harvest to each of my plants. I take requests, but if I don't get any, I abandon Hydroponics (and my super meme plant) after acquiring 50-100 "fruits" of my super meme plant and go and actually use those fruits myself (usually for pranking people with mutation toxin or the like).

How do you guys play botanist? What do you rush, focus on, use, etc.?

Shadowflame909 commented:
dude....

who has 1k credits to spend.

I wish all of this effort to create money gated barriers to botany was spread out upon other jobs.

I also wish my moar money PR got merged because botany having the lowest pay-out isn't gonna cut it anymore, since they have the most expensive items.

cacogen commented:
I make 100 potency bluespace tomatoes disguised as regular bananas and once
I have hundreds of them I leave them out for the crew while squashing my
own stash to randomly teleport around and throw them at people, teleporting
them and leaving a bluespace peel (You slip through spacetime!) and space
lubed tile behind.

I also used to make healing weed, but I haven't in a long time.

If I'm a lifebringer I get 100 potency space tobacco to offset the oxygen
damage on Lavaland from not having internals, and use 100 potency
cherrybombs to excavate Lavaland and 100 potency glowshrooms to light it.
Almost guaranteed to lose an arm. I've never had time before the shuttle
arrives to do the whole z-level.

On Tue, 21 Jan 2020 at 3:00 PM, ATH1909 <notifications@github.com> wrote:

> Maybe we all have different views on this because we play botanist very
> differently from each other? I, for one, almost never go for ambrosia gaia,
> even though some other people swear by it. I make a dispenser instead of
> stealing one, and will generally focus on making one super meme plant from
> the traits I find in strange seeds instead of several lesser meme plants. I
> never buy or use any fertilizers other than robust harvest unless I'm
> pumping my plant's stats, and even then, I make sure to apply at least a 1u
> dose of robust harvest to each of my plants. I take requests, but if I
> don't get any, I abandon Hydroponics (and my super meme plant) after
> acquiring 50-100 "fruits" of my super meme plant and go and actually use
> those fruits myself (usually for pranking people with mutation toxin or the
> like).
>
> How do you guys play botanist? What do you rush, focus on, use, etc.?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/48896?email_source=notifications&email_token=AF7NO6T25MZI4S4ULBIJL23Q6ZJKRA5CNFSM4KI5ORB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJOHX4Q#issuecomment-576486386>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7NO6U3EXGIURPKUDBBVOTQ6ZJKRANCNFSM4KI5ORBQ>
> .
>
-- 
Sent from my phone

ExcessiveUseOfCobblestone commented:
> But we do agree that this PR is a poor implementation of this idea, right? I mean, do you really think anyone is gonna buy a 30u bottle of ammonia for 1,000 credits?!

the numbers could use a review but I don't think that'll break the PR. I'm actually in agreement with the idea somewhat, although it's certainly not because there's some paper bin tricks one COULD use lol.

> Maybe we all have different views on this because we play botanist very differently from each other?

Possibly, but i'm going to treat the role like you are meant to provide food for the chef. Anything else is an added bonus and does not necessarily need to be catered to in order to do the job successfully.

cacogen commented:
> i'm going to treat the role like you are meant to provide food for the
chef

Because that's fun. That's why people pick the role.

On Tue, 21 Jan 2020 at 4:59 PM, moo <notifications@github.com> wrote:

> But we do agree that this PR is a poor implementation of this idea, right?
> I mean, do you really think anyone is gonna buy a 30u bottle of ammonia for
> 1,000 credits?!
>
> the numbers could use a review but I don't think that'll break the PR. I'm
> actually in agreement with the idea somewhat, although it's certainly not
> because there's some paper bin tricks one COULD use lol.
>
> Maybe we all have different views on this because we play botanist very
> differently from each other?
>
> Possibly, but i'm going to treat the role like you are meant to provide
> food for the chef. Anything else is an added bonus and does not necessarily
> need to be catered to in order to do the job successfully.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/tgstation/tgstation/pull/48896?email_source=notifications&email_token=AF7NO6XH6DQMJ7A4G4GLDGTQ6ZXLZA5CNFSM4KI5ORB2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEJONCVI#issuecomment-576508245>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AF7NO6R2ALRS24WF33VDHTDQ6ZXLZANCNFSM4KI5ORBQ>
> .
>
-- 
Sent from my phone

Geikkamir commented:
> Possibly, but i'm going to treat the role like you are meant to provide food for the chef. Anything else is an added bonus and does not necessarily need to be catered to in order to do the job successfully.

In other words designing the job to be played in the least fun and exciting way possible. Even _the chef themself_ barely cares about botany making food for them.

I don't think that focusing the job on the one aspect that literally no one is excited by is a particularly great approach.

Krysonism committed:
Adds back free ez nutrient, adjusts prices and improves sprite.

Krysonism commented:
I've added the free ez-nutrient bottles back after feedback.

I've also adjusted the prices and improved the sprite. see the updated picture.

Does the maintainers want the chem dispenser changes in this PR or a separate PR? Limiting the dispenser to just beakers and bottles also prevents the 250u spray bottle memes.

Krysonism committed:
custom_premium_price doesn't work for contraband.

FantasticFwoosh commented:
Couldnt you close this already, atomise it and start from the beginning again? The backpedalling is messy because you got it wrong the first time and you can't even gitbash/git cherrypick your commits because you etched botany changes all over them.

Added advantage of dealing with whatever pressure the maintainers are apparently giving you to make jerrycans compatible in a seperate PR. This should honestly be a contemporary example to teach other coders the pitfalls of not atomizing.

Ty-the-Smonk commented:
Honestly we should probably just have a system that lets us make custom fertilizer, make botany be a bit more proactive in making stuff.

ExcessiveUseOfCobblestone commented:
> Because that's fun. That's why people pick the role.

Let me clarify since I may have not conveyed it the way I wanted

Xenobio likes to make mob armies but all they get at the start is a 2 monkey cube boxes (1 for each computer) and are expected to replenish themselves. They don't start with 9 monkey cube boxes, 3 giving more extract and another 3 encouraging mutations. This is what I want for botany as a roundstart setup.

I didn't say I won't support features that enhance other forms of play, in fact I think the chem aspect is an opportunity we can capitalize on down the line. What I'm expecting is the roundstart equipment to support the only "obligation" botany has, and players branch out to do their own thing using this limited equipment.

> Honestly we should probably just have a system that lets us make custom fertilizer, make botany be a bit more proactive in making stuff.

not sure what you mean but you can make custom reagent combinations in plants then place them in the tray. If you're looking for something like just nutrients. There's definitely room to make the nutrients not so static (see add 1u of desired nutrient prior to harvest for ur effect lol).

81Denton commented:
I'm not convinced that nerfing the accessibility of EZ nutrient/left 4 zed/robust harvest will solve any of the issues that botany has, whether it's the uninteresting tier 1 plants or that mutagen is required to cut down on mutation RNG.

If anything, those 3 fertilizers are the only alternative to the annoying fetch-the-mutagen game that botanists play most rounds. And before someone gets the idea to nerf unstable mutagen - take a look at tier 1 plants and guess why barely anyone grows them.

The core issue with those is that growing food for the kitchen has zero incentives outside of RP since there are no rewards for it. Interactions in SS13 should always be engaging for both parties - as an example, miners deliver ores and in return receive mining points. 
If an interaction is only interesting for one party, it feels like serving burgers at a fast food joint.

ExcessiveUseOfCobblestone commented:
Not sure i agree with the accessibility argument considering all nutrients are easily accessible through the biogen. At the very least RH doesn't need to be given without planting some plants, it doesn't act as a mutagen-alt at all and only exists to boost your yields.
